libs/host/GarnetServer.cs

```diff
@@ -14,6 +14,11 @@
 
 namespace Garnet
 {
+    using MainStoreFunctions = StoreFunctions<SpanByte, SpanByte, SpanByteComparer, SpanByteRecordDisposer>;
+
```

Comment:
For symmetry add:
`using MainStoreAllocator = SpanByteAllocator<MainStoreFunctions>`